curator-framework/pom.xml

```diff
@@ -100,6 +100,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.3.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Import-Package>!org.apache.curator,*</Import-Package>
```

Comment:
There is no need to define the plugin again. The plugin is defined once in the parent pom and inherited from the rest of the modules.

To customize a modules configuration please set properties:
- `osgi.import.package`
- `osgi.export.package`